livekit/src/rtc_engine/peer_transport.rs

```diff
@@ -151,7 +203,25 @@ impl PeerTransport {
             return Ok(());
         }
 
-        let offer = self.peer_connection.create_offer(options).await?;
+        let mut offer = self.peer_connection.create_offer(options).await?;
+        let start_bitrate_kbps = DEFAULT_VP9_START_BITRATE_KBPS;
+        let sdp = offer.to_string();
+        // TODO, we should extend the codec support to AV1 ?
```

Comment:
worth testing if AV1 has this issue or not.. the JS logic is applying to AV1 as well